walkthroughs/week-3-server/examples/server-stats/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why was this file deleted?

Reply:
that was an accident and im not quite sure how to fix it @ljfranklin

Reply:
@zyelcrier try `git checkout master YOUR_FILE`. That tell git to update YOUR_FILE to match what it looks like on the master branch.